webapp/src/services/near-nep21-util.js

```diff
@@ -80,6 +81,23 @@ export async function gasCheck() {
   return false;
 }
 
+export function convertToReal( str ) {
+  const append = 25 - str.length;
+  if(append > 0) {
+    str = '0'.repeat(append) + str;
+  }
+  const pos = str.length - 24;
+  var res = [str.slice(0, pos), '.', str.slice(pos)].join('');
+  res = trimZeros(res);
```

Comment:
in this case it would be better to break `trimZeros` in 2 functions:
* `trim0Left`
* `trim0Right`

And us it:
```
var res = [trimLeft(str.slice(0, pos)), '.', trim0Right(str.slice(pos))].join('');
```

Reply:
but it for this case : 1e24
it will return : 1. then the code checks and remove '.'
this trimZeros does the same thing, it will only be repeated code if we make different functions

Reply:
trimZeros is also used in normailizeAccount

Reply:
OK, then you can leave it.